### ISSUE TITLE
bin/openshift-install destroy bootstrap --log-level debug shows FATAL error

### ISSUE THREAD

**Issue by praveenkumar** [openshift/installer#517]
# Version

```console
$ openshift-install version
bin/openshift-install v0.3.0-7-g123f6d1540c55d73eed6044e989d052ce50e8bf7
Terraform v0.11.9
```

# Platform (aws|libvirt|openshift): libvirt

# What happened?

```console
$ bin/openshift-install destroy bootstrap --log-level debug
INFO Using Terraform to destroy bootstrap resources... 
DEBUG Running &exec.Cmd{Path:"/home/prkumar/work/github/practice/go/src/github.com/openshift/installer/bin/terraform", Args:[]string{"/home/prkumar/work/github/practice/go/src/github.com/openshift/installer/bin/terraform", "init", "-input=false", "-no-color"}, Env:[]string(nil), Dir:"/tmp/openshift-install-258835319", Stdin:io.Reader(nil), Stdout:io.Writer(nil), Stderr:io.Writer(nil), ExtraFiles:[]*os.File(nil), SysProcAttr:(*syscall.SysProcAttr)(nil), Process:(*os.Process)(nil), ProcessState:(*os.ProcessState)(nil), ctx:context.Context(nil), lookPathErr:error(nil), finished:false, childFiles:[]*os.File(nil), closeAfterStart:[]io.Closer(nil), closeAfterWait:[]io.Closer(nil), goroutine:[]func() error(nil), errch:(chan error)(nil), waitDone:(chan struct {})(nil)}... 
Initializing modules...
- module.libvirt_base_volume
  Getting source "./volume"
- module.bootstrap
  Getting source "./bootstrap"

Initializing provider plugins...

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
DEBUG Running &exec.Cmd{Path:"/home/prkumar/work/github/practice/go/src/github.com/openshift/installer/bin/terraform", Args:[]string{"/home/prkumar/work/github/practice/go/src/github.com/openshift/installer/bin/terraform", "destroy", "-auto-approve", "-no-color", "-state=terraform.tfstate", "-target=module.bootstrap"}, Env:[]string(nil), Dir:"/tmp/openshift-install-258835319", Stdin:io.Reader(nil), Stdout:io.Writer(nil), Stderr:io.Writer(nil), ExtraFiles:[]*os.File(nil), SysProcAttr:(*syscall.SysProcAttr)(nil), Process:(*os.Process)(nil), ProcessState:(*os.ProcessState)(nil), ctx:context.Context(nil), lookPathErr:error(nil), finished:false, childFiles:[]*os.File(nil), closeAfterStart:[]io.Closer(nil), closeAfterWait:[]io.Closer(nil), goroutine:[]func() error(nil), errch:(chan error)(nil), waitDone:(chan struct {})(nil)}... 
libvirt_volume.coreos_base: Refreshing state... (ID: /var/lib/libvirt/images/coreos_base)
libvirt_ignition.bootstrap: Refreshing state... (ID: /var/lib/libvirt/images/bootstrap.ign;5bceb201-0396-d015-996d-707fa0678ee7)
data.libvirt_network_dns_host_template.etcds: Refreshing state...
data.libvirt_network_dns_host_template.bootstrap: Refreshing state...
data.libvirt_network_dns_host_template.masters: Refreshing state...
data.libvirt_network_dns_host_template.workers: Refreshing state...
libvirt_network.tectonic_net: Refreshing state... (ID: 1b80c1c9-a6d8-445a-aec9-6183b581185b)
libvirt_volume.bootstrap: Refreshing state... (ID: /var/lib/libvirt/images/bootstrap)

Error: Error refreshing state: 1 error(s) occurred:

* module.bootstrap.libvirt_ignition.bootstrap: 1 error(s) occurred:

* module.bootstrap.libvirt_ignition.bootstrap: libvirt_ignition.bootstrap: Error while retrieving remote volume: Can't retrieve volume /var/lib/libvirt/images/bootstrap.ign: virError(Code=50, Domain=18, Message='Storage volume not found: no storage vol with matching key /var/lib/libvirt/images/bootstrap.ign')

FATAL Error executing openshift-install: rename terraform.tfstate /tmp/openshift-install-258835319/terraform.tfstate: invalid cross-device link 
```

# What you expected to happen?

Shouldn't be any FATAL error

# How to reproduce it (as minimally and precisely as possible)?

```console
$ bin/openshift-install create cluster
<-- wait till cluster is up and running -->
$ bin/openshift-install destroy bootstrap --log-level debug
[...]
FATAL Error executing openshift-install: rename terraform.tfstate /tmp/openshift-install-258835319/terraform.tfstate: invalid cross-device link
```

**Comment by abhinavdahiya** [comment]
https://github.com/sirupsen/logrus/blob/master/logrus.go#L73-L92

So the FATAL is higher priority than DEBUG and --log-level debug allows all DEBUG and higher priority.

**Comment by wking** [comment]
`invalid cross-device link` was fixed by #509.
